joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -480,6 +496,7 @@ rclcpp_action::GoalResponse JointTrajectoryController::goalCB(
     goal->trajectory);
   traj_external_point_ptr_->update(traj_msg);
 
+  RCLCPP_INFO(lifecycle_node_->get_logger(), "accepted new action goal");
```

Comment:
`RCLCPP_DEBUG`?

Reply:
Hm I think this would be useful for the user to have as information. Let me know if you disagree.